### PR TITLE
Feature/breadcrumb

### PR DESCRIPTION
**Pull request by Julian0701**
### DEVELOP
- [x] KnowledgeManagementPage
- [x] KMDetailPage
- [x] KMArticle
- [x] KMPageBody
- [x] AuthorPage
- [x] Breadcrumb

### CHECK LIST

- [Naming convention](https://github.com/CAFECA-IO/WorkGuidelines/blob/main/technology/coding-convention/naming-convention.md) verification: checked
- Coding style verification: checked
- new Library: 0
- new Class / Component:  1
  - src/components/breadcrumb/breadcrumb.tsx
- new loop: 2
  - src/components/breadcrumb/breadcrumb.tsx line 10
  - src/components/km_page_body/km_page_body.tsx line 128
- high risk: 0
- new sql: 0

### CODE REVIEW

File: src/lib/posts.ts [hunk 1]
```diff
@@ -58,16 +58,19 @@ export async function getPosts(): Promise<IKnowledgeManagement[]> {
   return posts;
 }
 
-// Info: (20230721 - Julian) 抓取特定分類的文章
-export async function getPostsByCategory(category: string): Promise<IKnowledgeManagement[]> {
-  const posts: IKnowledgeManagement[] = [];
+export async function getCategorys(): Promise<string[]> {
```
Comment (arealclimber) [inline src/lib/posts.ts:61]: plural noun: categories

**Review by arealclimber — CHANGES_REQUESTED** [review]
發現變數錯字，再麻煩將categorys更改為categories，這是個人意見，如有任何疑問歡迎討論，謝謝

File: src/lib/posts.ts (hunk 1 above)
Comment (Julian0701) [reply src/lib/posts.ts]: 已修正

**Review by arealclimber — APPROVED** [review]

**Review by Luphia — APPROVED** [review]
- no bug found

**Pull request merged**